docs/_documentations/mdt-eclipse-importedprojects.md

```diff
@@ -24,6 +24,11 @@ Codewind is designed to develop cloud native microservices. Therefore, each proj
 * [Swift projects](#swift-projects)
 * [Generic Docker projects](#generic-docker-projects)
 
+### Binding Appsody projects
```

Comment:
We should add an entry to the bullet list above

Reply:
Got it. Yes, makes total sense. I'll add `Appsody project` as a bullet entry. Thanks!